contentcuration/contentcuration/utils/garbage_collect.py

```diff
@@ -8,6 +8,24 @@
 from contentcuration.models import ContentNode, File
 
 
+def clean_up_deleted_chefs():
+    """
+    Clean up all deleted chefs attached to the deleted chefs tree, including all
+    child nodes in that tree.
+
+    """
+    deleted_chefs_node, _new = ContentNode.objects.get_or_create(pk=settings.DELETED_CHEFS_ROOT_ID)
+    # we cannot use MPTT methods like get_descendants() or use tree_id because for performance reasons
+    # we are avoiding MPTT entirely.
```

Comment:
Good catch! Would this also delete the deleted chef root node?

Reply:
If I understand your question correctly, yes. We test that all nodes up to the chef's root node itself are deleted.